docs/README.md

```diff
@@ -0,0 +1,135 @@
+# Substrate Connect
```

Comment:
This file can be removed from the repository right?  It gets copied here by the deploy script

Reply:
Not yet. This will be removed once the landing page is ready( or else you will have nothing on landing page of github pages for now)